src/index.ts

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+import * as yargs from 'yargs';
+import { initProject as createProject, initProjectHere as createProjectHere } from './create-project';
+import { openDocs as docs } from './docs';
+
+// tslint:disable-next-line
+yargs
+  .usage('Usage: kak-cli <command>')
+  .version()
+  .command(
+    ['create-project <dir> [version]', 'cp <dir> [version]'],
+    'Create a new kakunin project in desired directory with optional kakunin version, default latest',
+    (args: yargs.Argv) => {
+      return yargs
+        .option('dir', {
+          demandOption: true,
+          string: true,
+        })
+        .option('version', {
+          demandOption: false,
+          string: true,
+        });
+    },
+    argv => {
+      console.log(`Starting to create new kakuknin project. This will take a while.`);
+      createProject(argv.dir)(argv.version);
```

Comment:
this one should take a single parameter - interface of configuration
this interface should look like this:
```
interface ProjectConfig {
  name: string,
  version: string|undefined,
  dir: string|undefined
}
```

based on it createProject function should either create a directory based on dir or cwd, update package.json with proper name, determine version based on configuration